libraries/lib-effects/MixAndRender.cpp

```diff
@@ -95,34 +95,12 @@ TrackListHolder MixAndRender(const TrackIterRange<const WaveTrack> &trackRange,
       oneinput = true;
    // only one input track (either 1 mono or one linked stereo pair)
 
-   // EmptyCopy carries over any interesting channel group information
-   // Maybe first is stereo, or it is mono, but some other input is stereo;
-   // therefore don't use WideEmptyCopy, but make and link a right channel as
-   // needed
-   // And reset pan and gain
-   auto mixLeft =
-      first->EmptyCopy(trackFactory->GetSampleBlockFactory(), false);
```

Comment:
You did what I asked, removing this and instead creating a "wide" track (as a TrackList) through the factory.

But now I see that this makes an unintended change of behavior.  The call of `WaveTrack::Init()` in `EmptyCopy` doesn't happen, and so certain information of `first` is no longer copied into the new track -- things such as the color, and whatever else in future might be attached to the `ChannelGroup` client data site.

So the new `Create` function should take `first` as another argument, which could default to nullptr, and from which it copies information as in `EmptyCopy`.

Or instead of that -- I think it's nicer -- add `std::optional<size_t> oNChannels = {}` as an argument in `EmptyCopy` and call that.